src/providers/VernaProvider.tsx

```diff
@@ -47,6 +46,10 @@ export interface VernaContextProps extends Pick<FormProps<unknown>, 'transformEr
     callback: (formData: unknown) => void,
   ) => (event: ISubmitEvent<FormData>, nativeEvent: FormEvent<HTMLFormElement>) => void;
   isEditor: boolean;
+  renderDropZone: React.FunctionComponent<DropZoneProps>;
+  renderFieldTemplate: React.FunctionComponent<FieldTemplateProps>;
+  renderSection: React.FunctionComponent<ObjectFieldTemplateProps>;
+  renderSubmitButton?: React.ReactNode;
```

Comment:
This is React component you should use a capitalized name then use it as a `<Component />`
```suggestion
  DropZone: React.FunctionComponent<DropZoneProps>;
  FieldTemplate: React.FunctionComponent<FieldTemplateProps>;
  Section: React.FunctionComponent<ObjectFieldTemplateProps>;
  SubmitButton?: React.ReactNode;
```